### PR TITLE
[#264] Fixed profile validator

### PR DESCRIPTION
**Pull request by aliaksandr-d**
#264 Fixed profile and secret validator: invalid property name and wrong template path to.
Validated profile & secret for Legion-demo.

### CODE REVIEW

File: deploy/ansible/create-cluster.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@
     - name: Validate profile yaml-file
       validator:
         source: "{{ lookup('env', 'PROFILES_PATH') or '../profiles' }}/{{ profile }}.yml"
-        template: "{{ lookup('env', 'PROFILES_PATH') or '../profiles' }}/profiles_template.yaml"
+        template: "../profiles/profiles_template.yaml"
```
Comment (kirillmakhonin) [inline deploy/ansible/create-cluster.yml]: I think that template should be parameterized as a source.

File: deploy/ansible/create-cluster.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@
     - name: Validate secret yaml-file
       validator:
         source: "{{ vars }}"
-        template: "{{ lookup('env', 'PROFILES_PATH') or '../profiles' }}/secrets_template.yaml"
+        template: "../profiles/secrets_template.yaml"
```
Comment (kirillmakhonin) [inline deploy/ansible/create-cluster.yml]: This too

File: deploy/ansible/create-cluster.yml (hunk 1 above)
Comment (aliaksandr-d) [reply deploy/ansible/create-cluster.yml]: for ADA and legion *PROFILES_PATH* has different values, due legion and ada profiles are located in different folders, BUT profile/secret templates should be single for any client and located in *profiles* folder in *legion* repo

**Comment by aliaksandr-d** [comment]
@kirillmakhonin please review, i've just applied fixes

**Comment by aliaksandr-d** [comment]
please test this

**Comment by aliaksandr-d** [comment]
test this please

**Comment by aliaksandr-d** [comment]
test this please

**Comment by alexander-semenets** [comment]
LGTM

**Pull request merged**